autoware_launch/config/perception/obstacle_segmentation/ground_segmentation/ground_segmentation.param.yaml

```diff
@@ -30,6 +30,7 @@
         detection_range_z_max: 2.5
         elevation_grid_mode: true
         use_recheck_ground_cluster: true
+        use_lowest_point: true
```

Comment:
Is this OK to set default value to be true?

Reply:
@YoshiRi Since current version is using [minimum height](https://github.com/autowarefoundation/autoware.universe/pull/6753/files#diff-2d2596f0217f44f1d4a1e131036aca53260aeeabf191d6cfcfd24f1b70971ddfL322) as default So I keep the parameter as `true` the same as current and will tune it where is neccessary on each product.